packages/flutter/lib/src/material/expand_icon.dart

```diff
@@ -116,7 +122,7 @@ class _ExpandIconState extends State<ExpandIcon> with SingleTickerProviderStateM
       onTapHint: widget.onPressed == null ? null : onTapHint,
       child: IconButton(
         padding: widget.padding,
-        color: theme.brightness == Brightness.dark ? Colors.white54 : Colors.black54,
+        color: widget.color ?? (theme.brightness == Brightness.dark ? Colors.white54 : Colors.black54),
```

Comment:
The Flutter style has changed since this was written: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-if-chains-or-or-with-enum-values